code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -424,4 +433,30 @@ GLOBAL_LIST_EMPTY(ts_spiderling_list)
 				var/pc_of_max_per_second = round(((hp_points_per_second / maxHealth) * 100), 0.1)
 				stat(null, "Regeneration: [ltext]: <font color='[lcolor]'>[num2text(pc_of_max_per_second)]% of health per second</font>")
 
-
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/DoRemoteView()
+	if(client && (client.eye != client.mob))
+		reset_perspective()
+		return
+	if(health != maxHealth)
+		to_chat(src, "<span class='warning'>You must be at full health to do this!</span>")
+		return
```

Comment:
Huh?
So if the princess nest is raided, but she escapes with only a scratch - she can no longer provide her leadership to the hive? That sounds very weird..
What's the motivation here?

Reply:
Queens regenerate very fast. If she only takes a scratch, she'll have fully healed before she even reaches her new nest.
If she's severely wounded, then I think it makes sense for the pain/damage to block her viewing ability for a few minutes.
I also want to discourage wounded queens from using it again until they're in a nest and have had a chance to heal up.